Dockerfile

```diff
@@ -48,17 +54,17 @@ RUN        true \
 # not trigger additional install as part of the build
 # in the following step.
 COPY       . /calypso/
-RUN        touch node_modules
+RUN        true \
+           && find . -not -path './node_modules/*' -print0 | xargs -0 chown calypso:calypso \
+           && touch node_modules \
+           && true
 
 # Build the final layer
 #
 # This contains built environments of Calypso. It will
 # change any time any of the Calypso source-code changes.
+USER       calypso
 ARG        commit_sha=(unknown)
-RUN        true \
-           && CALYPSO_ENV=production COMMIT_SHA=$commit_sha npm run build \
-           && find . -not -path './node_modules/*' -print0 | xargs -0 chown nobody \
-           && true
-
-USER       nobody
+RUN        CALYPSO_ENV=production COMMIT_SHA=$commit_sha npm run build
+RUN        find . -not -user calypso -or -not -group calypso | wc -l
```

Comment:
Useful to verify ownership is correct. Should produce output like the following:

```
Step 16/17 : RUN        find . -not -user calypso -or -not -group calypso | wc -l
 ---> Running in b39b6b133107
0
```

Prune before merge!